pallets/loans/src/mock.rs

```diff
@@ -129,17 +132,37 @@ impl pallet_balances::Config for Runtime {
     type WeightInfo = pallet_balances::weights::SubstrateWeight<Runtime>;
 }
 
+lazy_static! {
+    pub static ref PRICE_ORACLE: Mutex<HashMap<CurrencyId, Option<PriceDetail>>> = {
+        Mutex::new(
+            vec![DOT, KSM, BTC, USDT, XDOT]
```

Comment:
BTC will not be supported until further discussion, we should remove it from code and test cases.

Reply:
All the currency types in the lending market will be listed by the governance module. Current markets are more of a demonstration.